erigon-lib/downloader/downloadercfg/downloadercfg.go

```diff
@@ -65,7 +65,7 @@ func Default() *torrent.ClientConfig {
 	// better don't increase because erigon periodically producing "new seedable files" - and adding them to downloader.
 	// it must not impact chain tip sync - so, limit resources to minimum by default.
 	// but when downloader is started as a separated process - rise it to max
-	//torrentConfig.PieceHashersPerTorrent = max(1, runtime.NumCPU()-1)
+	torrentConfig.PieceHashersPerTorrent = max(1, runtime.NumCPU()-2)
```

Comment:
because there is no global limiter for hashers - on non-nvme drives it may crash with `> 10K threads created` panic. because all threads will wait for overloaded/just_slow disk (and go creating dedicated thread for each IO-blocked goroutine).